symforce/slam/imu_preintegration/preintegrated_imu.py

```diff
@@ -0,0 +1,105 @@
+# ----------------------------------------------------------------------------
+# SymForce - Copyright 2022, Skydio, Inc.
+# This source code is under the Apache 2.0 license found in the LICENSE file.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+
+from sym.factors.imu_preintegration_update import imu_preintegration_update
+
+
+class PreintegratedImu:
```

Comment:
@varunagrawal have the updates you mention landed in gtsam develop?

Reply:
Yes they have!

Reply:
@varunagrawal Oh great. I am nose down on getting this port going and comparing with current gtsam develop. Perhaps I can ping you with a question or two. =)

Reply:
Absolutely! More than happy to help and see this feature landed in Symforce.

Reply:
Glad y'all are interested :)

We're also currently pushing on getting this usable as a factor in the optimizer and getting it merged, it might be worth coordinating and sharing the work (@bradley-solliday-skydio is working on this right now)

Reply:
@bradley-solliday-skydio i would love to discuss. I am working through unit testing the tangent pre-integration piece against gtsam, and finding some differences. Lmk.

Reply:
any update on this feature?

Reply:
Hey Hans!  @bradley-solliday-skydio is actively working on finishing this up and we should be able to merge it soon